lib/usdUfe/utils/mergePrims.h

```diff
@@ -45,4 +45,14 @@ bool mergePrims(
     const PXR_NS::SdfPath&        dstPath,
     const MergePrimsOptions&      options);
 
-} // namespace MayaUsdUtils
+//! \brief merge prims with default options, but no verbosity.
+USDUFE_PUBLIC
+bool mergePrims(
+    const PXR_NS::UsdStageRefPtr& srcStage,
+    const PXR_NS::SdfLayerRefPtr& srcLayer,
+    const PXR_NS::SdfPath&        srcPath,
+    const PXR_NS::UsdStageRefPtr& dstStage,
+    const PXR_NS::SdfLayerRefPtr& dstLayer,
+    const PXR_NS::SdfPath&        dstPath);
```

Comment:
New helper function which doesn't take a MergePrimOptions - it just uses defaults, with verbosity = None.